suite-common/wallet-graph/src/graphUtils.ts

```diff
@@ -514,3 +517,32 @@ export const getLineGraphAllTimeStepInMinutes = (
     // to prevent max URL length error, HTTP status 414, from Blockbook (timestamps are sent to Blockbook with HTTP GET)
     return lineGraphStepInMinutes.year * differenceYears;
 };
+
+export const getTimeFrameConfiguration = (
+    timeFrame: LineGraphTimeFrameValues,
+    endOfRangeDate: Date,
+    minutesBackToStartOfRange: number,
+) => {
+    const stepInMinutes =
+        timeSwitchItems[timeFrame]?.stepInMinutes ??
+        getLineGraphAllTimeStepInMinutes(endOfRangeDate, minutesBackToStartOfRange);
+
+    return {
+        ...timeSwitchItems[timeFrame],
+        valueBackInMinutes: minutesBackToStartOfRange,
+        stepInMinutes,
+    };
+};
+
+export const sortTimeFrameItemsByTimeAsc = (
+    accountBalanceMovements: LineGraphTimeFrameItemAccountBalance[],
+) => accountBalanceMovements.sort((a, b) => a.time - b.time);
```

Comment:
Please do not use that mutable built-in shit sort, use `A.sort` instead

Reply:
https://github.com/trezor/trezor-suite/pull/6585/commits/90a58371ff675ff2f27b4477b9081f78bfacd50b